types/stripe/index.d.ts

```diff
@@ -1,170 +1,123 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare const Stripe: StripeStatic;
-
-interface StripeStatic {
-    applePay: StripeApplePay;
-    setPublishableKey(key: string): void;
-    validateCardNumber(cardNumber: string): boolean;
-    validateExpiry(month: string, year: string): boolean;
-    validateCVC(cardCVC: string): boolean;
-    cardType(cardNumber: string): StripeCardDataBrand;
-    getToken(token: string, responseHandler: (status: number, response: StripeCardTokenResponse) => void): void;
-    card: StripeCard;
-    createToken(data: StripeCardTokenData, responseHandler: (status: number, response: StripeCardTokenResponse) => void): void;
-    bankAccount: StripeBankAccount;
-}
-
-interface StripeCardTokenData {
-    number: string;
-    exp_month?: number;
-    exp_year?: number;
-    exp?: string;
-    cvc?: string;
-    name?: string;
-    address_line1?: string;
-    address_line2?: string;
-    address_city?: string;
-    address_state?: string;
-    address_zip?: string;
-    address_country?: string;
-}
-
-interface StripeTokenResponse {
-    id: string;
-    client_ip: string;
-    created: number;
-    livemode: boolean;
-    object: string;
-    type: string;
-    used: boolean;
-    error?: StripeError;
-}
-
-interface StripeCardTokenResponse extends StripeTokenResponse {
-    card: StripeCard;
-}
-
-interface StripeError {
-    type: string;
-    code: string;
-    message: string;
-    param?: string;
-}
-
-type StripeCardDataBrand = 'Visa' | 'American Express' | 'MasterCard' | 'Discover' | 'JCB' | 'Diners Club' | 'Unknown';
-
-type StripeCardDataFunding = 'credit' | 'debit' | 'prepaid' | 'unknown';
-
-interface StripeCard {
-    object: string;
-    last4: string;
-    exp_month: number;
-    exp_year: number;
-    country?: string;
-    name?: string;
-    address_line1?: string;
-    address_line2?: string;
-    address_city?: string;
-    address_state?: string;
-    address_zip?: string;
-    address_country?: string;
-    brand?: StripeCardDataBrand;
-    funding?: StripeCardDataFunding;
-    createToken(data: StripeCardTokenData, responseHandler: (status: number, response: StripeCardTokenResponse) => void): void;
-    validateCardNumber(cardNumber: string): boolean;
-    validateExpiry(month: string, year: string): boolean;
-    validateCVC(cardCVC: string): boolean;
-}
-
-interface StripeBankAccount {
-    createToken(params: StripeBankTokenParams, stripeResponseHandler: (status: number, response: StripeBankTokenResponse) => void): void;
-    validateRoutingNumber(routingNumber: number | string, countryCode: string): boolean;
-    validateAccountNumber(accountNumber: number | string, countryCode: string): boolean;
-}
-
-interface StripeBankTokenParams {
-    country: string;
-    currency: string;
-    account_number: number | string;
-    routing_number?: number | string;
-    account_holder_name: string;
-    account_holder_type: string;
+/// <reference types="stripe/v2" />
```

Comment:
in general, after you copy the exiting to v2, which you have done, you want to add, and not remove.